### PR TITLE
RDS-1341 - Fix explorers

### PR DESCRIPTION
**Pull request by roszpun**

### CODE REVIEW

File: src/data/networks.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
   },
   "sei": {
     "rpcUrl": "https://evm-rpc.sei-apis.com",
-    "explorerUrl": "https://seiexplorer.com",
+    "explorerUrl": "https://www.mintscan.io/sei",
```
Comment (bohmac) [inline src/data/networks.json:95]: should be https://seitrace.com/?chain=pacific-1
according to SEI docs
https://www.sei.io/ecosystem

File: src/data/networks.json [hunk 2]
```diff
@@ -155,14 +155,14 @@
   },
   "real": {
     "rpcUrl": "https://real.drpc.org",
-    "explorerUrl": "https://explorer.real.com",
+    "explorerUrl": "https://testnet-explorer.plumenetwork.xyz",
```
Comment (bohmac) [inline src/data/networks.json:158]: https://explorer.re.al/

File: src/data/networks.json [hunk 3]
```diff
@@ -218,7 +218,7 @@
   },
   "berachainTestnet": {
     "rpcUrl": "https://bera-testnet.nodeinfra.com",
-    "explorerUrl": "https://testnet-explorer.berachain.com",
+    "explorerUrl": "https://80084.testnet.routescan.io/",
```
Comment (bohmac) [inline src/data/networks.json:221]: https://bartio.beratrail.io/

File: src/data/networks.json (hunk 1 above)
Comment (roszpun) [reply src/data/networks.json:95]: But none of the addresses we use for the SEI does not appear in this explorer

Comment (roszpun) [reply src/data/networks.json:95]: nvm it works.

**Review by cehali — APPROVED** [review]

**Review by rpiszczatowski — APPROVED** [review]

**Pull request merged**